node_modules/col-whiteboards/lib/api.js

```diff
@@ -908,12 +908,9 @@ var scheduleThumbnail = function(whiteboardId) {
  *
  * @param  {Number}   assetId               The id of the asset for which a preview image has been generated
  * @param  {String}   imageUrl              The URL where the generated image can be downloaded
- * @param  {Object}   dimensions            The dimensions of the PNG
- * @param  {Number}   dimensions.width      The width of the PNG file
- * @param  {Number}   dimensions.height     The height of the PNG file
- * @api private
+ * @param  {Number}   width                 The width of the generated image
```

Comment:
Because we replace the image in other whiteboards but we want to retain the size/scale